src/components/TextInput/baseTextInputPropTypes.js

```diff
@@ -46,9 +46,6 @@ const propTypes = {
     /** Saves a draft of the input value when used in a form */
     shouldSaveDraft: PropTypes.bool,
 
-    /** Character limit for the input */
-    maxLength: PropTypes.number,
```

Comment:
`maxLength` and `hint` will be used once [TextInput](https://github.com/Expensify/App/issues/7522) is refactored. We should probably keep it here and find another solution to this issue.

Reply:
Can you add it once you refactor it?

Reply:
Sure, but wouldn't we run into this same issue once a contributor refactors the component?

Reply:
Discussed 1:1 and we will remove it for now and reintroduce these props on the actual implementation.